addon/Config.jsm

```diff
@@ -22,6 +22,10 @@ var config = {
     "variation": {
       "name": "eTLD+1",
     },
+    // True if the addon is run for a simulation.
+    "isSimulation": true,
+    // Path containing the RAPPOR simulator.
+    "rapporPath": "/Users/arodriguez/src/rappor/",
```

Comment:
Make this an empty string and call it rapporSimulatorPath.

Reply:
Done